src/double_integrator/value_iteration.py

```diff
@@ -109,6 +112,26 @@ def draw(iteration, mesh, cost_to_go, policy):
 surf = ax2.plot_surface(Q, Qdot, Pi, rstride=1, cstride=1,
                         cmap=cm.jet)
 
-plt.show()
+# animate the resulting policy.
+builder = DiagramBuilder()
+plant = builder.AddSystem(DoubleIntegrator())
+logger = builder.AddSystem(SignalLogger(2))
+builder.Connect(plant.get_output_port(0), logger.get_input_port(0))
```

Comment:
n.b. these two lines can be 
logger = LogOutput(plant.get_output_port(0), builder)

Reply:
Done.